### PR TITLE
Update to shift to HTTPS

### PR DESCRIPTION
**Pull request by anshumgargdd**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by cswatt — APPROVED** [review]

**Comment by anshumgargdd** [comment]
@cswatt can you make the change to the url used in the destination to the appropropriate one for each data center? e.g http-intake.logs.us5.datadoghq.com for US5?
I wasn't sure how to do that.

File: content/en/integrations/syslog_ng.md [hunk 1]
```diff
@@ -69,8 +69,14 @@ Configure Syslog-ng to gather logs from your host, containers, & services.
     ...
 
     # For Datadog platform:
-    template DatadogFormat { template("YOURAPIKEY <${PRI}>1 ${ISODATE} ${HOST:--} ${PROGRAM:--} ${PID:--} ${MSGID:--} ${SDATA:--} $MSG\n"); };
-    destination d_datadog { tcp("intake.logs.datadoghq.com" port(10514) template(DatadogFormat)); };
+    destination d_datadog {
+      http(
+          url("https://http-intake.logs.datadoghq.com/api/v2/logs?ddsource=<SOURCE>&ddtags=<TAG_1:VALUE_1,TAG_2:VALUE_2>")
```
Comment (Stoovles) [inline content/en/integrations/syslog_ng.md:74]: From the looks of it this is an update for the us3 region, so it should point to the us3 intake:
`https://http-intake.logs.us3.datadoghq.com/api/v2/logs`

File: content/en/integrations/syslog_ng.md [hunk 2]
```diff
@@ -85,35 +91,7 @@ Configure Syslog-ng to gather logs from your host, containers, & services.
     log { source(s_src); source(s_files); destination(d_datadog); };
     ```
```
Comment (Stoovles) [inline content/en/integrations/syslog_ng.md:92]: If we're going to remove the bit about TLS encryption we should probably include a link to the `d_http` syslog-ng module to give customers some agency over making changes to the config if desired

Something like:

> More information about http destination parameters and possibilities are available in the [syslog-ng Open Source Edition Administration Guide](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.26/administration-guide/38).

**Comment by cswatt** [comment]
@anshumgargdd @Stoovles pushed a change so that the code will reflect the endpoint in the site selector

File: content/en/integrations/syslog_ng.md (hunk 2 above)
Comment (anshumgargdd) [reply content/en/integrations/syslog_ng.md:92]: yeah we can incorporate that.

**Pull request merged**